crates/pop-common/src/templates/extractor.rs

```diff
@@ -1,26 +1,33 @@
 // SPDX-License-Identifier: GPL-3.0
 
-use anyhow::Result;
 use std::{fs, io, path::Path};
 
+use anyhow::Result;
+
 /// Extracts the specified template files from the repository folder to the target folder.
 ///
 /// # Arguments
-/// * `template_name` - The name of the template to extract.
+/// * `template_file` - The name of the template to extract.
 /// * `repo_folder` - The path to the repository folder containing the template.
```

Comment:
Directory and folder are used interchangeably, where perhaps we should stick to one. This obviously isnt the goal of this PR, but it make make sense to move towards a standard term for any code that is changed.

My suggestion would be directory.

Reply:
@al3mart if you are working in this change, do it directly here: https://github.com/r0gue-io/pop-cli/pull/283 so will be applied for the OZ PR too.

Reply:
https://github.com/r0gue-io/pop-cli/pull/286